ui/index.html

```diff
@@ -35,6 +35,13 @@
           .addEventListener('click', peerWeb.search.refresh)
       })
     </script>
+    <script>
+      var ipcRenderer = require('electron').ipcRenderer;
+      ipcRenderer.on('global-status-reply',(event,arg)=>{
+        console.log(arg);//Testing only
```

Comment:
This is super sexy!

I'll wire this up to a dashboard sometime this week :heart: